docs/jackson.md

```diff
@@ -55,7 +55,7 @@ Or with Gradle:
 compile 'org.eclipse.collections:eclipse-collections:10.4.0'
 ```
 
-9.2.0 is the current Eclipse Collections version at time of writing. Check out [the maven repo](https://search.maven.org/search?q=g:org.eclipse.collections%20AND%20a:eclipse-collections&core=gav) for newer versions.
+10.4.0 is the current Eclipse Collections version at time of writing. Check out [the maven repo](https://search.maven.org/search?q=g:org.eclipse.collections%20AND%20a:eclipse-collections&core=gav) for newer versions.
```

Comment:
I would suggest deleting the version section altogether. It will make it 1 more thing to update for every release. Currently, there are too many things to remember to update, so I would not want to add more.

Reply:
Same comment about the Jackson version. We should just have a since Jackson version `so and so` Eclipse Collections is supported. It will be a nightmare to keep the doc synced with the Jackson versions.

Reply:
I removed the version section for Eclipse Collections and referenced the GitHub releases for Maven and Gradle Coordinates. @yawkat @nikhilnanivadekar can you take a look and provide any additional feedback.

Reply:
Looks fine to me.